guides/plugins/apps/administration/flow-builder/add-custom-flow-actions-from-app-system.md

```diff
@@ -0,0 +1,243 @@
+# Add custom flow actions via App System
+
+
```

Comment:
Could you bring this into more of a "guide" structure - https://github.com/shopware/docs/blob/master/resources/guidelines/documentation/templates/guide-documentation-template.md

e.g. Explain what you're going to do + add a "prerequisites" and "next steps" section

Reply:
Please use this as a reference - https://github.com/shopware/docs/pull/579/files#diff-0d6ef1b2b9c6863269597da0d12bd5cdbe8f322137e55f3b6d6e2e129ab870e6

Also, mention the version from which this feature is available as in the `{% hint style="info" %}` section of the referenced guide, please.

Reply:
When you use codeblocks like `{% code title="manifest.xml %}, please also provide a filename.

Otherwise, just use triple backticks ` ``` ` to escape your code.